### PR TITLE
Add validation checks for transcript model

### PR DESCRIPTION
**Pull request by nassersaazi**
Closes [this issue](https://github.com/adamjonas/transcription-queue/issues/10)

### CODE REVIEW

File: api/app/controllers/transcript.controller.js [hunk 1]
```diff
@@ -6,18 +6,38 @@ const Op = db.Sequelize.Op;
...
+
+    return transcriptHash;
+  }
 
   // Create a Transcript
   const transcript = {
-    // We have to add title because for some reason having just content makes an update to an existing record insted of inserting a new one
     originalContent: req.body.originalContent,
-    content: req.body.content
+    content:req.body.content,
```
Comment (adamjonas) [inline api/app/controllers/transcript.controller.js:39]:
```suggestion
    content: req.body.content,
```

File: init.sql [hunk 2]
```diff
@@ -190,23 +236,30 @@ ALTER TABLE ONLY public.users ALTER COLUMN id SET DEFAULT nextval('public.users_
...
... So that's great from the privacy point of view. Schnor also enables new features, new contracting features, which were impossible before. So in this case, we want to have different contracts or puppies. Execute only if an off-chain Oracle creates the corresponding signature. So say Oracle signs B that A. That signature allows signature one to be created, which spends pub key one, but not for pub key two, and not for pub key three. If the Oracle signs for event B, the second pub key can execute. If Oracle signs for C, the third. So this allows us to bring trusted off-chain information onto on-chain script execution. Again, in a way which is observable by the on-chain observer, they just look like regular single-spend transactions. So this can part of the toolkit is around taproot. Taproot means we can hide Bitcoin script in a very nice way. So let's first consider how Bitcoin scripts work today. This is, I think, recognized in the HLC. HLCs are used to forward payments and lightning, and an HLC needs to encapsulate quite a bit of, you know, spending logic. The first one is the success case. So the forwarding payment has been successfully delivered. This is the part of the execution
```
Comment (adamjonas) [inline init.sql:259]: what is this?

File: init.sql [hunk 3]
```diff
@@ -215,30 +268,35 @@ COPY public.transcripts (id, content, "originalContent", status, "archivedAt", "
 --
 
 COPY public.users (id, "githubUsername", "authToken", permissions, "archivedAt", "createdAt", "updatedAt") FROM stdin;
```
Comment (adamjonas) [inline init.sql:270]: so there will be no user's in the seed? how will we grant admin permissions to anyone from a fresh seed?

File: init.sql [hunk 4]
```diff
@@ -257,6 +315,22 @@ ALTER TABLE ONLY public.transcripts
...
+    ADD CONSTRAINT "transcripts_transcriptHash_key" UNIQUE ("transcriptHash");
+
+
+--
+-- Name: tutorials tutorials_pkey; Type: CONSTRAINT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.tutorials
+    ADD CONSTRAINT tutorials_pkey PRIMARY KEY (id);
+
```
Comment (adamjonas) [inline init.sql:332]: what are tutorials?

File: init.sql (hunk 3 above)
Comment (nassersaazi) [reply init.sql:224]: i have updated this to include users from [here](https://queue.btctranscripts.com/api/users)

File: init.sql (hunk 4 above)
Comment (nassersaazi) [reply init.sql]: this was data used to test during initial set up. i have since removed it

File: init.sql (hunk 2 above)
Comment (nassersaazi) [reply init.sql]: i have cleaned up that to use data from [here](https://queue.btctranscripts.com/api/transcripts)

[the remark this answers is not in the document]
Comment (nassersaazi) [reply init.sql]: i have cleaned up that to use data from [here](https://queue.btctranscripts.com/api/transcripts)

File: .gitignore [hunk 5]
```diff
@@ -1,4 +1,5 @@
 node_modules
+api/bitcointranscripts
```
Comment (adamjonas) [inline .gitignore:2]: what is this?

File: api/server.js [hunk 6]
```diff
@@ -2,9 +2,9 @@ const express = require("express"),
   bodyParser = require("body-parser"),
   swaggerJsdoc = require("swagger-jsdoc"),
   swaggerUi = require("swagger-ui-express");
-// const dotenv = require("dotenv");
+ const dotenv = require("dotenv");
 
-// dotenv.config();
+ dotenv.config();
```
Comment (adamjonas) [inline api/server.js:7]: where are we using these?

File: .gitignore (hunk 5 above)
Comment (nassersaazi) [reply .gitignore]: these were some test files for PR opening. i've removed the line in the latest commit together with some redundant comments

**Pull request merged**